lib/Common/Memory/Recycler.cpp

```diff
@@ -242,7 +242,9 @@ Recycler::Recycler(AllocationPolicyManager * policyManager, IdleDecommitPageAllo
 #ifdef HEAP_ENUMERATION_VALIDATION
     ,pfPostHeapEnumScanCallback(nullptr)
 #endif
+#ifndef TARGET_CHAKRACORE
```

Comment:
I will but `NTBUILD` does not sound like this is for ChakraFull build. It reminds me WinNT instead..